### ISSUE TITLE
[Desktop] Major performance regression in the latest stable public release.

### ISSUE THREAD

**Issue by davinci26** [brave/brave-browser#11453]
## Description 

Brave Browser consumes *absurd* amount of CPU for the the GPU Process and the main Browser Process. (GPU Process 60% CPU) according to Task manager.

As you can understand this makes Brave pretty much unsable. 

## Steps to Reproduce
Not 100% repro.

## Actual result:
CPU spike.

## Expected result:
Expect to have similar performance to other Chromium based browsers.

## Reproduces how often: 
Reproduces fairly often but not deterministically

## Brave version (brave://version info)

```
Brave | 1.12.114 Chromium: 84.0.4147.135 (Official Build) (64-bit)
-- | --
Revision | c42bd09b3f24da1698d71d3b4f57402137163566-refs/branch-heads/4147@{#1102}
OS | macOS Version 10.14.6 (Build 18G2022)
JavaScript | V8 8.4.371.23
```
## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by Martina-Neumayer** [comment]
Look under brave:gpu at the hardware acceleration status and also for some missing libs (this was in my case the cause of the enormous cpu/gpu usage).

**Comment by rebron** [comment]
@davinci26 Can you update to 1.13.82 and let us know if you're still seeing performance issues.

**Comment by jvarghe** [comment]
I can confirm that I have noticed this problem too. In fact, I came here specifically to complain about it. After I updated my browser last Friday (August 28, 2020), my laptop started to sound like a leafblower when ever I opened up Brave. 

Right now, Brave is consuming upwards of 90% of my CPU and has been for hours. I've tested Brave for an entire week and its been like this non-stop. As best as I can tell, the problem start with a Brave release made in the week of August 21, 2020 and August 28, 2020. Before then, Brave was the lightest browser on my computer, but now it is the heaviest and most resource consuming. 

@rebron: I'm running Brave v1.13.82. It also has this CPU performance bug.

**Comment by Martina-Neumayer** [comment]
Look at those issues: #11412 and #9824

**Comment by bsclifton** [comment]
@Martina-Neumayer we did recently fix an issue on 1.16 (Nightly) which would be great if you wanted to try it. We'll be moving this over to Beta channel next week

Here's the issue we fixed which I think might give you better performance 😄 
https://github.com/brave/brave-browser/issues/11504

**Comment by Martina-Neumayer** [comment]
@bsclifton I have now 1.15.67 Chromium: 86.0.4240.55 version and seems that HW works fine.
I don't see any error messages about missing libs.

One thing that remains is something about "unknown libva error" in the "brave:gpu" panel:

"Log Messages
[1713:1713:0930/091652.912501:ERROR:vaapi_wrapper.cc(546)] : vaInitialize failed: unknown libva error"

But this does not affects the whole browser performance in the hardware accelerated mode.

**Comment by rebron** [comment]
@davinci26 @jvarghe It looks like we may have this fixed in #11504. That fix is currently in Beta, if you have a chance we'd love if you can see if that version is running ok. Otherwise it's looking like end of October for that fix to reach you.

**Comment by jvarghe** [comment]
@rebron I don't know if this bugfix has hit stable yet, but it must have. Brave is back to its normal behaviour. It's flirtation with turning computers into leaf-blowers has ended! 

rebron, thanks for putting together the fix.
